Packed ticks in a loop instead of only for the first

OrBin opened the pull request:
Fixes #73

kenorb commented:
Now, for some reason it's time outing after 10mins which didn't happen before.
Suspected line is [this](https://github.com/FX31337/FX-BT-Scripts/blob/master/Makefile#L72).
Maybe it's in the wrong dir and downloaded files doesn't exist, or something.
You can test on local by running `make test` I guess.

OrBin commented:
@kenorb, I see. Currently working on it. That's weird, because this exact commit passed build in my repository, and the base doesn't seem to be different in yours.

OrBin commented:
OK, now I've fixed it. If you don't run `make clean` before `make test`, something goes wrong there. Hence, I added in `before_script` section of the travis configuration file.
